front/src/utils/config.js

```diff
@@ -12,6 +12,10 @@ const config = {
     productCard: {
       width: 350,
       height: 200
+    },
+    categoryCard: {
+      width: 350,
```

Comment:
Please rename to imageWidth
Same for height to imageHeight
Can you please also do it for product card (don't forget to modify the component)

Reply:
Than you for advice! Done!